src/Pim/Bundle/ReferenceDataBundle/Doctrine/ORM/Repository/ReferenceDataRepository.php

```diff
@@ -90,6 +90,18 @@ public function findOneByIdentifier($identifier)
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function findByIdentifiers(array $referenceDataCodes)
```

Comment:
as you use this method to get only codes, you should rename it to `findCodeByIdentifiers()` and add a `->select('r.code')` (btw why do you call `$this->getAlias()` ? define a letter is more readable imho)